src/clients/twitch.ts

```diff
@@ -47,7 +51,7 @@ function registerTwitch(): void
             {
                 'clientId': configFile.T2D_CLIENT_ID,
                 'clientSecret': configFile.T2D_SECRET,
-                onRefresh: async newTokenData => await fs.writeFile('./tokens.json', JSON.stringify(newTokenData, null, 4), 'utf-8')
+                onRefresh: async newTokenData => configFile.DEV_TWITCH_TOKEN ? await fs.writeFile('./tokens.json', JSON.stringify(newTokenData, null, 4), 'utf-8') : null
```

Comment:
The glorious land of jank solutions brings forth beauty <3

My co-workers don't like it when I do this, however I like it, so this gets a pass XD

Reply:
look. if it works it works!